packages/snaps-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- **BREAKING:** Add `snap_resolveInterface` RPC method to the `SnapInterfaceController` ([#2509](https://github.com/metamask/snaps/pull/2509))
```

Comment:
I believe this change is only breaking in `snaps-rpc-methods` which was already released. Will check with @GuillaumeRx though.

Reply:
Confirmed that this is not breaking.

Reply:
Thanks for clearing that up! Fixed here: https://github.com/MetaMask/snaps/pull/2589/commits/867aa2a909a5dd1f827a1b6fac190e9851358314